tracer/test/Datadog.Trace.TestHelpers/SpanTagAssertion.cs

```diff
@@ -28,6 +28,7 @@ internal SpanTagAssertion(Result result, IDictionary<string, T> tags)
             .IsOptional("version")
             .IsOptional("_dd.p.dm")   // "decision maker", but contains the sampling mechanism
             .IsOptional("_dd.p.tid")  // contains the upper 64 bits of a 128-bit trace id
+            .IsOptional("_dd.parent_id") // Contains the 16 length hex decoded value of `p`
```

Comment:
Instead of "contain the value of `p`", can you explain what that value is? Something like, "the last seen parent id".

Reply:
Also in the PR title and description 😄

Reply:
Addressed in: https://github.com/DataDog/dd-trace-dotnet/pull/5721/commits/42164062f5114055ccb6a5c71c20031a6438476c